src/main/java/org/jabref/gui/util/ViewModelTableRowFactory.java

```diff
@@ -104,97 +112,128 @@ public ViewModelTableRowFactory<S> withTooltip(Callback<S, String> toTooltip) {
         return this;
     }
 
+    public ViewModelTableRowFactory<S> withPseudoClass(PseudoClass pseudoClass, Callback<S, ObservableValue<Boolean>> toCondition) {
+        this.pseudoClasses.putIfAbsent(pseudoClass, toCondition);
+        return this;
+    }
+
     @Override
     public TableRow<S> call(TableView<S> tableView) {
-        TableRow<S> row = new TableRow<>();
-
-        if (toTooltip != null) {
-            String tooltipText = toTooltip.call(row.getItem());
-            if (StringUtil.isNotBlank(tooltipText)) {
-                row.setTooltip(new Tooltip(tooltipText));
```

Comment:
In the old code we set them to the row

Reply:
Fixed in [`be04ce9` (#11510)](https://github.com/JabRef/jabref/pull/11510/commits/be04ce97c02953ac1b56065eb7b2d2ce7efaa12d).
I think the `ViewModelTreeTableRowFactory` and `ViewModelListCellFactory` classes should also be updated because they register all events in the `updateItem` method.